### PR TITLE
fix classpath entries to address build failure from withinn eclipse IDE

### PR DESCRIPTION
**Pull request by nprigour**
Some classpath entries are missing or refere to wrong versions in .classpath file.

How did this passed unnoticed! Don't you use eclipse for building? :)
 
Signed-off-by: Nikolaos Pringouris <nprigour@gmail.com>

### CODE REVIEW

File: plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -272,7 +271,7 @@ Bundle-ClassPath: .,
  lib/jfreechart-1.0.13.jar,
  lib/jfreechart-experimental-1.0.13.jar,
  lib/jfreechart-swt-1.0.13.jar,
- lib/jgridshift-1.1.jar,
+ lib/jgridshift-core-1.2.jar,
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF]: Looks like we missed these ;)

File: plugins/org.locationtech.udig.libs/build.properties [hunk 2]
```diff
@@ -282,7 +282,9 @@ bin.includes = .,\
                lib/xml-commons-resolver-1.2.jar,\
                lib/xmlrpc-client-3.0.jar,\
                lib/xmlrpc-common-3.0.jar,\
-               lib/xpp3_min-1.1.4c.jar
+               lib/xpp3_min-1.1.4c.jar,\
+               lib/jgridshift-core-1.2.jar,\
+               lib/jackson-core-2.10.1.jar
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.libs/build.properties]: 👍

File: plugins/org.locationtech.udig.libs/.classpath [hunk 3]
```diff
@@ -171,7 +170,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-1.0.13.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-experimental-1.0.13.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jfreechart-swt-1.0.13.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/jgridshift-1.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jgridshift-core-1.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jgt-oms3-0.7.6.jar"/>
```
Comment (fgdrf) [inline plugins/org.locationtech.udig.libs/.classpath]: Well, IMHO I would't add this file (in addition to .settings/ folders) to source-repository anymore. M2e plugins within Eclipse IDE would created it from MF-Files /pom.xml files of project. I recently used it and got rid of this eclipse-ide-specific files.

Comment (nprigour) [reply plugins/org.locationtech.udig.libs/.classpath]: Maybe we should accept this PR and then you create a seperate one addressing removal of all .classpath files and .settings folder

Comment (fgdrf) [reply plugins/org.locationtech.udig.libs/.classpath]: Was just waiting for the travis build feedback ;)

Thanks for the fix!

**Pull request merged**